prov/cxi/configure.m4

```diff
@@ -46,11 +46,11 @@ AC_DEFUN([FI_CXI_CONFIGURE],[
 		[
 			AC_CHECK_HEADER(cxi_prov_hw.h,
 				[],
-				[AC_MSG_ERROR("cxi_prov_hw.h not found")])
+				[cxi_happy=0])
```

Comment:
Please squash this into the first commit.

Reply:
I mean the entire commit, not this specific change.

Reply:
I plan to. I want to gather as much feedback as possible without resquashing and losing comments (it happens sometimes).